src/prg_quantumdynamics_mod.F90

```diff
@@ -300,12 +220,15 @@ end subroutine prg_kick_density_bml
   !! \param A matrix in first index of the commutator
   !! \param B matrix in second index of the commutator
   !! \param C output matrix that is the commutator of A and B
-  subroutine prg_commutate_bml(A,B,C)
+  !! \param thresh the BML threshold
+  !!
+  subroutine prg_commutate_bml(A,B,C,thresh)
```

Comment:
@nicolasbock It is written as "future work" in the bml manual, so I think there are plans to do it. But I agree, it seems like something that would be good to have in bml instead. I am not currently using this module. Do you want me to delete it and just plan on it being written into bml later?